src/scikit_build_core/resources/_editable_redirect.py

```diff
@@ -180,5 +192,7 @@ def install(
             verbose,
             build_options or [],
             install_options or [],
+            DIR,
+            os.path.join(DIR, install_dir) if install_dir else DIR,
```

Comment:
I am not sure exactly why `DIR` was passed like optional variable, but for now, let's keep it like that. The other suggestion is to move this path construction to the `ScikitBuildRedirectingFinder` constructor

```suggestion
            install_dir=install_dir,
```

Probably you can simplify:
```python
        self.install_dir = os.path.join(DIR, install_dir or "")
```

Although, you might want to first gate it by an `os.path.isabs()`. @henryiii, why are we not using `pathlib.Path` for these?

Reply:
Ah yeah, you did mention that last time as well. :+1: